applications/ParticleMechanicsApplication/custom_utilities/mpm_boundary_rotation_utility.h

```diff
@@ -179,26 +179,10 @@ class MPMBoundaryRotationUtility: public CoordinateTransformationUtils<TLocalMat
 	void ApplySlipCondition(TLocalVectorType& rLocalVector,
 			GeometryType& rGeometry) const override
 	{
-		if (rLocalVector.size() > 0)
-		{
-			for(unsigned int itNode = 0; itNode < rGeometry.PointsNumber(); ++itNode)
-			{
-				if( this->IsSlip(rGeometry[itNode]) )
-				{
-					// We fix the first momentum dof (normal component) for each rotated block
-					unsigned int j = itNode * this->GetBlockSize(); // +1
-
-					// Get the displacement of the boundary mesh, this does not assume that the mesh is moving.
-					// If the mesh is moving, need to consider the displacement of the moving mesh into account.
-					const array_1d<double,3> & displacement = rGeometry[itNode].FastGetSolutionStepValue(DISPLACEMENT);
-					array_1d<double,3> rN = rGeometry[itNode].FastGetSolutionStepValue(NORMAL);
-					this->Normalize(rN);
-
-					rLocalVector[j] = inner_prod(rN,displacement);
-
-				}
-			}
-		}
+        // creates an empty dummy matrix to pass into the 'full' ApplySlipCondition -- this dummy matrix is
+        // ignored, effectively only updating the RHS
+        TLocalMatrixType dummyMatrix = TLocalMatrixType();
```

Comment:
```suggestion
        TLocalMatrixType dummyMatrix;
```

this should be enough